tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "include": [
-    "src",
+    "src", 
+    "src/identities/*.ts", 
```

Comment:
why this necessary? Would be better `src/*`

Reply:
Indeed it's not necessary: `src` seems to work